src/AppRouter.js

```diff
@@ -10,7 +10,7 @@ export default function AppRouter() {
       <Switch>
         <Route exact path="/" component={HomePage} />
         <Route path="/add-item" component={AddItem} />
-        <Route path="/list" component={List} />
+        <Route path="/list/:token" component={List} />
```

Comment:
Help me understand why we need the token in the route when we have it available in `localStorage`?

Reply:
That can be taken out. I was trying to get around the state not getting saved to localStorage.

Reply:
I'm removing all of those. Honestly, I got tired. I knew they needed to go 😃